httpFilePaths.cmake

```diff
@@ -18,10 +18,5 @@ set( HTTP_INCLUDE_PUBLIC_DIRS
 set( HTTP_INCLUDE_THIRD_PARTY_DIRS
```

Comment:
Would a user ever need these or these are just for our tests? If later, better not to have them. Same for HTTP_THIRD_PARTY_SOURCES.

Reply:
It is used for the demos also

Reply:
Hmm I am thinking about this more, and the third party source is private. So I will group it under the HTTP_PRIVATE_SOURCE